content/he/es6.md

```diff
@@ -1,20 +1,22 @@
-# ES6 on io.js
+<div dir="rtl" lang="he">
+# ES6 ב io.js 
```

Comment:
It looks like the markdown is being ignored here. Markdown gets inconsistent once you try to wrap divs around things.

When I put together the "list of translations" recently I included a right-to-left indicator for us to be able to use internally to make template-level fixes for RTL languages, see https://github.com/iojs/website/blob/master/source/project.json#L12